ffi/tests/ctests/src/tests.c

```diff
@@ -0,0 +1,307 @@
+#include <stdlib.h>
+#include <stdbool.h>
+#include <string.h>
+
+#include <wirefilter.h>
+
+extern void rust_assert(bool check, const char *msg);
+
+static wirefilter_externally_allocated_str_t wirefilter_string(const char *s) {
+	return (wirefilter_externally_allocated_str_t){s, strlen(s)};
```

Comment:
Ugh, apparently, MSVC considers this "an extension" and throws due to "extra warnings" being enabled...

```
src/tests.c(10): warning C4204: nonstandard extension used: non-constant aggregate initializer
```

Can you please change it (again) to a separate variable and initialization so that our headers work anywhere?

```c
wirefilter_externally_allocated_str_t str;
str.data = s;
str.length = strlen(s);
return str;
```

Reply:
Thats what i thought :) This syntax is some kind of c++-ism

Reply:
It's been standardised in C99 (although I much prefer named properties like `(wirefilter_externally_allocated_str_t) { .data = ..., .length = ... }`) but MSVC is not quite there yet...

Reply:
I fixed it :+1: